recipes/cxxtest/cxxtestgen

```diff
@@ -0,0 +1,20 @@
+#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
This should be changed to `#!/usr/bin/env python`.

Reply:
Done.